docs/conf.py

```diff
@@ -0,0 +1,36 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Project information -----------------------------------------------------
+
+project = u"evops"
+copyright = u"2021, Dmitriy Jarosh"
```

Comment:
I think its better to mention Pavel here)

Reply:
Sure, done